src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -463,12 +468,16 @@ type FieldTypeFormatterRenderFunctionPartial = (data: EventData) => React.ReactN
 export function getFieldFormatter(
   field: string,
   meta: MetaType
-): FieldTypeFormatterRenderFunctionPartial {
+): FieldTypeFormatterRenderFunction {
   const fieldName = getAggregateAlias(field);
   const fieldType = meta[fieldName];
 
   if (FIELD_FORMATTERS.hasOwnProperty(fieldType)) {
-    return partial(FIELD_FORMATTERS[fieldType].renderFunc, fieldName);
+    return function (props) {
+      return FIELD_FORMATTERS[fieldType].renderFunc({field: fieldName, ...props});
```

Comment:
Removing the use of partial here makes this function inconsistent with `getFieldRenderer` since that returns a `FieldTypeFormatterRenderFunctionPartial` which has support for an extra `baggage` parameter. Would using baggage to pass the `align` parameter be better here?

Reply:
Unfortunately,  `baggage` isn't present for the field formatters. For special fields, `baggage` is a required parameter. 



`(props, baggage) => React.ReactNode` is a signature that works for both the field formatters and special fields. Note that `baggage` is ignored for field formatters since only the first argument is used.